Description: don't remote-link SourceForge's logo
 This let SourceForge track everyone who reads these docs.  As the image is
 not freely licensed, let's leave only a piece of text.
--- htdig-3.2.0b6.orig/htdoc/FAQ.html
+++ htdig-3.2.0b6/htdoc/FAQ.html
@@ -2585,6 +2585,6 @@ find /path/to/your/document/root -type f
 	  	Last modified: $Date: 2004/05/28 13:15:16 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
   </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/author.html
+++ htdig-3.2.0b6/htdoc/author.html
@@ -34,6 +34,6 @@
 	Last modified: $Date: 2004/05/28 13:15:18 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
   </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/bugs.html
+++ htdig-3.2.0b6/htdoc/bugs.html
@@ -50,6 +50,6 @@
 	Last modified: $Date: 2004/05/28 13:15:18 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
   </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/isp.html
+++ htdig-3.2.0b6/htdoc/isp.html
@@ -81,7 +81,7 @@
 	Last modified: $Date: 2004/05/28 13:15:19 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
 
 </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/mailing.html
+++ htdig-3.2.0b6/htdoc/mailing.html
@@ -55,6 +55,6 @@
 	Last modified: $Date: 2004/05/28 13:15:19 $
 <br>
     <a href="http://sourceforge.net/">
-      <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+      SourceForge</a>
   </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/main.html
+++ htdig-3.2.0b6/htdoc/main.html
@@ -103,6 +103,6 @@
 	Last modified: $Date: 2004/05/28 13:15:19 $
 <br>
     <a href="http://sourceforge.net/">
-      <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+      SourceForge</a>
   </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/rundig.html
+++ htdig-3.2.0b6/htdoc/rundig.html
@@ -184,7 +184,7 @@
 	Last modified: $Date: 2004/06/12 13:39:13 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
 
   </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/running.html
+++ htdig-3.2.0b6/htdoc/running.html
@@ -131,7 +131,7 @@
 	Last modified: $Date: 2004/05/28 13:15:19 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
 
   </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/uses.html
+++ htdig-3.2.0b6/htdoc/uses.html
@@ -646,7 +646,7 @@
 	Last modified: $Date: 2004/05/28 13:15:19 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
 
 </body>
 </html>
--- htdig-3.2.0b6.orig/htdoc/where.html
+++ htdig-3.2.0b6/htdoc/where.html
@@ -121,6 +121,6 @@
 <br> 
 
    <a href="http://sourceforge.net/"> 
-<img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a> 
+SourceForge</a> 
   </body>
 </html>
--- htdig-3.2.0b6.orig/test/htdocs/set1/bad_local.htm
+++ htdig-3.2.0b6/test/htdocs/set1/bad_local.htm
@@ -51,7 +51,7 @@
 	Last modified: $Date: 2004/01/18 12:54:15 $
 <br> 
     <a href="http://sourceforge.net/"> 
-          <img src="http://sourceforge.net/sflogo.php?group_id=4593&amp;type=1" width="88" height="31" border="0" alt="SourceForge Logo"></a>
+          SourceForge</a>
 
   </body>
 </html>
